core/templates/domain/objects/NumberWithUnitsObjectFactory.ts

```diff
@@ -99,8 +99,8 @@ export class NumberWithUnits {
   }
 
   toMathjsCompatibleString(): string {
-    var numberWithUnitsString: string = '';
-    var unitsString: string = (new UnitsObjectFactory()).fromList(
+    let numberWithUnitsString: string = '';
```

Comment:
Do we need to mention the type if we're already initializing it? Won't it infer the type?

Reply:
Yes, done.